docs/src/reconciling-local-changes.md

```diff
@@ -43,14 +43,23 @@ invalidating the data in `config.yml`.
 
 ### Non-destructive, non-blocking changes
 Additive changes are often accommodated with no immediate operational
-issues, Consider manually adding an extension. Because TPA does not make
-destructive changes, the extension will not be removed when `tpaexec
-deploy` is next run.
+issues. Consider manually adding a user. The new user will continue to
+exist and cause no issues with TPA at all. You may prefer to manage the
+user through TPA in which case you can declare it in `config.yml` but
+the existence of a manually-added user will cause no operational issues.
+
+Some manual additions can have more nuanced effects. Consider manually
```

Comment:
"Consider manually adding an extension" could be seen as a request. Suggest "Take the example of an extension which has been manually added".